doc/tutorials/plutus-playground.rst

```diff
@@ -0,0 +1,222 @@
+.. highlight:: haskell
+.. _basic_playground_tutorial:
+
+Compiling and testing a Plutus App in the Plutus Playground
+===========================================================
+
+The Plutus Playground consists of a code editor and a simulator.
+
+.. figure:: images/playground.png
+
+  Plutus Playground (code editor)
+
+We can write Plutus Apps in the code editor and test them in the simulator.
+The Plutus Playground includes a number of sample apps.
+In this tutorial we are going to run through two of the sample apps.
+
+Running the "Hello, World" app
+------------------------------
+
+When we first open the Plutus Playground, the code editor is populated with the "Vesting" app.
+To load the "Hello, World" app, click the link in the top row.
+This app is much simpler than the vesting app.
+In fact, the app itself only consists of a single line, and a type annotation:
+
+.. literalinclude:: HelloWorldApp.hs
+    :start-after: BLOCK1
+    :end-before: BLOCK2
+
+Compiling the app
+^^^^^^^^^^^^^^^^^
+
+To compile the app, click the "Compile" button.
+The feedback line at the bottom of the code editor changes to "Compiling...".
+When the app has been compiled without errors, the feedback message changes to "Compilation successful", and the "Simulate" button is active.
+
+Running the first simulation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now that the app has been compiled, we can run it in the simulated environment. 
+Click the "Simulate" button.
+The code editor disappears and the simulator opens.
+
+.. figure:: images/playground-simulator.png
+
+  Plutus Playground (simulator)
+
+We can use the simulator to define complex scenarios with multiple agents trading and communicating over the Cardano blockchain.
+For the "Hello, World" app however we don't need to change anything in the simulator.
+We can click "Evaluate" right away.
+This sends the simulation to the server, alongside the source code of our app.
+When the server is done running the simulation, the transactions pane opens, showing the outcome of the simulation.
+
+Interpreting the results of the first simulation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The "Transactions" pane has four sections: Blockchain, final balances, logs and trace.
+The blockchain section shows all the transactions that were produced during the simulation.
+
+.. figure:: images/playground-transactions.png
+
+  Simulated blockchain with a single transaction
+
+
+Each simulation is run on a fresh blockchain.
+The "Hello, World" simulation only results in a single transaction.
+This is the initial transaction that distributes some funds to the simulated agents.
+We can click the box that says "Slot 0, Tx 0" to examine the transaction.
+Now there are two new sections visible.
+The first of the new sections show the inputs and outputs of the selected transaction, and some other details.
```

Comment:
```suggestion
The first of the new sections shows the inputs and outputs of the selected transaction, and some other details.
```

Reply:
Also, could you perhaps add an actual screenshot of these newly displayed sections that are being described here?